usefulUtilities/airAndWhiteboardDraw/README.md

```diff
@@ -12,37 +12,37 @@ This utility combines the best features of the whiteboard and the draw app. Any
 
 ## Releases
 
-SHA [](https://github.com/highfidelity/hifi-content/commits/)
+SHA [71386f1](https://github.com/highfidelity/hifi-content/pull/455/commits/71386f1)
```

Comment:
Please change all of these links to match the format https://github.com/highfidelity/hifi-content/commits/<commit SHA>